dev-util/spec-cleaner/files/spec-cleaner-1.0.6-pytest-runner.patch

```diff
@@ -5,6 +5,7 @@
      keywords=['SUSE', 'RPM', '.spec', 'cleaner'],
```

Comment:
You've amended this to the wrong commit. You can help yourself with `git reflog` to find the previous version of this commit to restore.